docs/content/en/docs/getting-started/install-docker.md

```diff
@@ -23,15 +23,39 @@ The easiest way to start experimenting with Tetragon is to run it via Docker
 using the released container images.
 
 ```shell
-docker run --name tetragon-container --rm --pull always \
+docker run -d --name tetragon --rm --pull always \
     --pid=host --cgroupns=host --privileged             \
     -v /sys/kernel/btf/vmlinux:/var/lib/tetragon/btf    \
-    quay.io/cilium/tetragon-ci:latest
+    quay.io/cilium/tetragon:latest
 ```
 
-This will start Tetragon in a privileged container. Priviliges are required
-to load and attach BPF programs. See Installation section for more details.
+This will start Tetragon in a privileged container running in the background.
+Running Tetragon as a privileged container is required to load and attach BPF
+programs. See the [Installation and Configuration]({{< ref "/docs/installation" >}})
+section for more details.
 
-## What's Next
+## Run demo application
+
+To explore Tetragon it is helpful to have a sample workload. You can use Cilium’s
+demo application with [this Docker Compose file](https://github.com/cilium/tetragon/blob/main/examples/quickstart/docker-compose.yaml),
+but any workload would work equally well:
+
+```shell
+curl -LO https://github.com/cilium/tetragon/raw/main/examples/quickstart/docker-compose.yaml
+docker compose -f docker-compose.yaml up -d
+```
+
+You can use `docker container ls` to verify that the containers are up and
+running. It might take a short amount of time for the container images to
+download.
+
+```shell
```

Comment:
forgot to add to remove shell

Reply:
So you want no language designator here, just the backticks?

Reply:
yes, it's a nit again, but it makes the input look different than the output, this is subtle but this is the convention we have in the doc:
<img width="699" alt="image" src="https://github.com/user-attachments/assets/a9c395db-f76e-4087-b20c-db1350fe222c">